### PR TITLE
Add release script

### PR DESCRIPTION
**Pull request by kzh**
This PR adds a minimal release script. It produces the plugin jar and uploads it to our google cloud storage bucket.

It's a WIP. Git tagging will likely be added. The release process isn't determined at the moment. I'm also considering doing all of this in a CI process.

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#
+# This script is to be run in the root folder.
+# It is to be run with two arguments:
+#
+# ./scripts/release.sh <OLD VERSION> <NEW VERSION>
+#
+# ex: ./scripts/release.sh 1.1.0 1.2.0
+
+atlas-package
+gsutil mv gs://sourcegraph-for-bitbucket-server/latest.jar gs://sourcegraph-for-bitbucket-server/${0}.jar
```
Comment (keegancsmith) [inline scripts/release.sh]: you probably meant `${1}` and `${2}`. `${0}` is the executable name.

File: scripts/release.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+gsutil mv ./target/sourcegraph-bitbucket-${1}.jar gs://sourcegraph-for-bitbucket-server/latest.jar
```
Comment (keegancsmith) [inline scripts/release.sh]: do you mean `gsutil cp`? if you wanted a mv then you should just upload directly to this target?

**Review by keegancsmith — CHANGES_REQUESTED** [review]

File: scripts/release.sh (hunk 1 above)
Comment (kzh) [reply scripts/release.sh]: Fixed!

File: scripts/release.sh (hunk 2 above)
Comment (kzh) [reply scripts/release.sh]: Fixed!

File: scripts/release.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+# The dependencies are:
+# 1) atlassian sdk (for building jar)
+# 2) gcloud cli setup with sourcegraph-dev project (for uploading jar)
+#
+# This script is to be run in the root folder.
+#
+# ./scripts/release.sh <OLD VERSION> <NEW VERSION>
+#
+# ex: ./scripts/release.sh 1.1.0 1.2.0
+
```
Comment (mrnugget) [inline scripts/release.sh:15]: I think this should have at the very least a `set -e` at the top so it doesn't continue on if anything here fails.

**Review by mrnugget — CHANGES_REQUESTED** [review]

**Comment by tsenart** [comment]
Next steps?

**Comment by kzh** [comment]
I think this PR is mergeable in its current state.

**Comment by ryanslade** [comment]
PTAL

I've just used this to release the latest version so it's in a mergeable state I think

**Comment by ryanslade** [comment]
The concerns above have been taken care of. I'm going to merge this so that it's not forgotten and we can make improvements if we need to.

**Pull request merged**